### DISCUSSION TITLE
Can we apply the Terser plugin for module federation chunks?

### DISCUSSION THREAD

**Discussion by nihil-pro** [module-federation/module-federation-examples#3629] [Q&A]
This is an example of mf output code, i want to format it. How can i do that? Thanks
`let BrandModel = class BrandModel {
            [brandModelSetTypeKey] = Brand_model_model_1.BrandModelModel;
            id = crypto.randomUUID();
            name = "";
            brandModelSet = [];
            constructor() {
                (0,
                mobx_0.makeObservable)(this, {
                    id: mobx_0.observable,
                    name: mobx_0.observable,
                    brandModelSet: mobx_0.observable
                })
            }
            setName(name) {
                (0,
                mobx_0.runInAction)((()=>this.name = name))
            }
            setModels(models) {
                (0,
                mobx_0.runInAction)((()=>this.brandModelSet = models))
            }
            addModel(model) {
                (0,
                mobx_0.runInAction)((()=>this.brandModelSet.push(model)))
            }
        }
        ;
        BrandModel = __decorate([utils_2.AutoBind], BrandModel)`

**Comment by ScriptedAlchemy** [comment]
Optimization options in the bundler. It will optimize like anything else in the build. It's just another chunk.
